pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -108,111 +107,69 @@ Ea8/B6hPatJ0ES8q/HO3X8IVQwVs1n3aAr0im0/T+Xc=
-
-
-```html
-openssl aes-256-cbc -md md5 -d -k $PASSWORD -in cert.pem.enc -out cert.pem
-openssl aes-256-cbc -md md5 -d -k $PASSWORD -in privkey.pem.enc -out privkey.pem
-```
-
-
-## Determine which URLs will be signed 
-
-You will need to create a URL pattern that defines which documents should be signed. It is critical that private content, such as personalized information should not be signed, to avoid sending misleading or incorrect content.
-
-For performance purposes, the packager should only be passed valid AMP documents as input. Some invalid AMP documents are fine if needed, but you should avoid sending all traffic through the packager. 
-
-
-## Deploy packager to a staging server
-
-You should set Signed Exchanges up on a staging server first, to verify a correct setup, before migrating to production to ensure proper implementation. We recommend using [`amppackager`](https://github.com/ampproject/amppackager/blob/master/README.md) and its built-in web server. 
-
-Follow instructions [here to set up `amppackager` for your site](https://github.com/ampproject/amppackager/blob/master/README.md). 
+Follow instructions [here to set up `amppackager` for your site](https://github.com/ampproject/amppackager/blob/master/README.md).
 
 [tip type="read-on"]
-The [`packager.js`](https://github.com/ampproject/docs/blob/future/platform/lib/routers/packager.js) file for `amp.dev` demonstrates Signed Exchange support setup.
+See [`packager.js`](https://github.com/ampproject/docs/blob/future/platform/lib/routers/packager.js) (used by `amp.dev`) for an example of the server-side changes you will need to make to route the required requests `amppkg`.
 [/tip]
 
-NOTE: Alternatively, you can use the command-line clients [`transform`](https://github.com/ampproject/amppackager/blob/master/transformer/README.md) and [`gen-signedexchange`](https://github.com/WICG/webpackage/tree/master/go/signedexchange), and handle version negotiation and certificate management tasks by hand. This should only be implemented if `amppackager` does not fit for your production environment. You will need to serve with the appropriate content-types (`application/signed-exchange;v=b3` and `application/cert-chain+cbor`). 
+### Testing
 
-Test that your staging site returns the content with the added MIME type `application/signed-exchange` when provided with the correct request headers. In the below example, replace `staging.example.com` with your staging server.
+Verify that your staging site returns the content with the added MIME type `application/signed-exchange` when provided with the correct request headers. In the below example, replace `staging.example.com` with your staging server.
 
-
-```html
-$ curl -si -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/ | less
+```sh
+$ curl -si -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/
```

Comment:
I typically leave `less` in example commands like this because I don't want control characters to get output to people's terminals -- could be insecure. I think, in fact, `curl` will not output binary data directly to the terminal -- you didn't experience this? Anyway, your call; just wanted to explain my reasoning.

(Ditto below.)

Reply:
That's fair, added `| less` back.

(Also, for me, curl is happy to output to the terminal. Maybe because OS X? Because the first n bytes are ASCII headers, and don't trip a heuristic?)